deployments/cloudfoundry/README.md

```diff
@@ -8,4 +12,8 @@ In this configuration, the Collector will run in the same container as the app.
 ### Bosh Release
 
 This is a Bosh release of the Splunk Collector. This deploys the Collector to the PCF
-environment as a standalone deployment.
\ No newline at end of file
+environment as a standalone deployment.
+
+### Tanzu Tile
+
+Packaged release of the Splunk Collector that can be integrated into the Ops Manager.
```

Comment:
Is this a definition of "Tanzu Tile"? Is there any more info you can provide or maybe screenshots?

Reply:
I've attempted to update the description with more information. I've also included a couple screenshots in the README of the main tile, as well as the view for changing configuration options.